index.html

```diff
@@ -8,9 +8,99 @@
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <body>
+
+  <!-- Copy and paste your HTML from the first UI project here -->
+  <nav>
+    <img class="mobileLogo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+
+  <a href='services.html'>Services</a>
+  <a href='#'>Product</a>
+  <a href='#'>Vision</a>
+  <a href='#'>Features</a>
+  <a href='#'>About</a>
+  <a href='#'>Contact</a>
+
+  <img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+</nav>
+
+<section class="jumbo">
```

Comment:
```suggestion
</header>
<section class="jumbo">
```

Reply:
Adds a header tag (which most pages will have) and removes the img from inside nav and places it on as a sibling of nav (both children of header)